README.md

```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Chia-Jou's portfolio and SPS projects.
 
 To get started:
```

Comment:
Why not remove this part?

[NO ACTION REQUIRED] This is a sample comment. A reviewer will add such comments to your code to ask for clarifications or suggest some changes